tools/run_tests/performance/loadtest_config.py

```diff
@@ -118,17 +118,72 @@ def gen_run_indices(runs_per_test: int) -> Iterable[str]:
         yield index_fmt.format(i)
 
 
+def scenario_name(base_name, client_channels, server_threads, offered_load):
+    elements = [base_name]
+    if client_channels:
+        elements.append(f'{client_channels}channels')
+    if server_threads:
+        elements.append(f'{server_threads}threads')
+    if offered_load:
+        elements.append(f'{offered_load}load')
+    return '_'.join(elements)
+
+
+def scenario_transform_function(
+    client_channels: int, server_threads: int, offered_loads: Iterable[int]
+) -> Optional[Callable[[Iterable[Mapping[str, Any]]], Iterable[Mapping[str,
+                                                                       Any]]]]:
+    """Returns a transform to be applied to a list of scenarios."""
+    if not any([client_channels, server_threads, len(offered_loads)]):
```

Comment:
Nit: I would use () instead of [] since it is immutable, but it is not important.

Reply:
I appreciate you point this out :)